### PR TITLE
Add some more tools

### LINKED ISSUE #253: Add to resources: csslayout.io
URL: https://csslayout.io/

Description: a collection of popular layouts and patterns made with CSS

For: Web

### LINKED ISSUE #254: Add to resources: responsively.app
URL: https://responsively.app/

Description: Preview multiple mobile/desktop versions of a website at the same time

For: Web

### LINKED ISSUE #255: Add to resources: Bash script template
Title: Bash Scripting Template

URL: https://github.com/ralish/bash-script-template

Description: A Bash scripting template incorporating best practices & several useful functions.

For: Bash

### LINKED ISSUE #256: Add to resources: https://github.com/mfaerevaag/wd
Title: warp directory

URL: https://github.com/mfaerevaag/wd

Description: lets you jump to custom directories in zsh, without using typing a long path with `cd`.

For: zsh

### PR DESCRIPTION
**Pull request by carpenterd777**
- Added wd to resources, fixes #256
- Moved tools from resources to tools article markdown
- Added tools route
- Added tools button to sidebar menu
- Added bash script template, fixes #255
- Add responsively, fixes #254
- Add csslayout.io, fixes #253

### CODE REVIEW

File: src/Router.js [hunk 1]
```diff
@@ -331,6 +331,13 @@ const ROUTES = [
 			import(/* webpackChunkName: "quick-slides" */ "./views/Slides/QuickSlides.js")
 		),
 	},
+    {
+        path: "/tools",
```
Comment (dyc3) [inline src/Router.js:335]:
```suggestion
        path: "/guide/tools",
```

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -65,6 +65,10 @@ const Header = () => {
 			title: "Pimp My Terminal",
 			route: "/event/pimp-my-terminal",
 		},
+        {
+            title: "Tools",
+            route: "/tools",
```
Comment (dyc3) [inline src/components/Header/Header.jsx:70]:
```suggestion
            route: "/guide/tools",
```

**Review by dyc3 — CHANGES_REQUESTED** [review]

**Comment by dyc3** [comment]
bruh how did you get a merge conflict

**Comment by carpenterd777** [comment]
im havin a moment over here

File: src/articles/guides/tools.md [hunk 3]
```diff
@@ -32,6 +34,14 @@ Familiarizing yourself with powerful tools will streamline your workflow and hel
 
 -   [fugitive](https://github.com/tpope/vim-fugitive) - A plugin for Vim that wraps Git to allow easier command usage
 
+## Shell Scripting
```
Comment (dyc3) [inline src/articles/guides/tools.md:37]:
```suggestion
## Bash
```

**Review by dyc3 — APPROVED** [review]

File: src/articles/guides/tools.md (hunk 3 above)
Comment (carpenterd777) [reply src/articles/guides/tools.md]: if you write a script for bash, will it run for other shells?

**Pull request merged**